angular-route/templates/_.client.route.js

```diff
@@ -0,0 +1,5 @@
+$stateProvider.
+  state('<%= slugifiedName %>', {
+    url: '/<%= slugifiedRoutePath %>',
+    templateUrl: 'modules/<%= slugifiedModuleName %>/client/views/<%= slugifiedViewName %>.client.view.html'
+  }).
```

Comment:
@ilanbiala It ends with a . exactly because it's a partial. Look up `_.client.routes.js` - that's a template for routes file, which should contain many of them. If the file already exists, this partial is taken instead, and inserted into an already existing routes file. See `index.js`, line 89 and below.